modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/UserPersonalSitePermissions.java

```diff
@@ -1,278 +0,0 @@
-
-import com.liferay.application.list.PanelApp;
-import com.liferay.application.list.constants.PanelCategoryKeys;
-import com.liferay.osgi.util.ServiceTrackerFactory;
-import com.liferay.petra.string.StringBundler;
-import com.liferay.portal.kernel.exception.PortalException;
-import com.liferay.portal.kernel.log.Log;
-import com.liferay.portal.kernel.log.LogFactoryUtil;
-import com.liferay.portal.kernel.model.Company;
-import com.liferay.portal.kernel.model.Group;
-import com.liferay.portal.kernel.model.Portlet;
-import com.liferay.portal.kernel.model.ResourceConstants;
-import com.liferay.portal.kernel.model.Role;
-import com.liferay.portal.kernel.model.role.RoleConstants;
-import com.liferay.portal.kernel.security.permission.ResourceActionsUtil;
-import com.liferay.portal.kernel.service.CompanyLocalService;
-import com.liferay.portal.kernel.service.GroupLocalService;
-import com.liferay.portal.kernel.service.PortletLocalService;
-import com.liferay.portal.kernel.service.ResourcePermissionLocalService;
-import com.liferay.portal.kernel.service.RoleLocalService;
-import com.liferay.portal.kernel.util.Validator;
-
-import java.util.List;
-
-import org.osgi.framework.BundleContext;
-import org.osgi.framework.ServiceReference;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-import org.osgi.util.tracker.ServiceTracker;
-import org.osgi.util.tracker.ServiceTrackerCustomizer;
-
-/**
- * @author Tomas Polesovsky
- */
-@Component(service = UserPersonalSitePermissions.class)
-public class UserPersonalSitePermissions {
-
-	public void initPermissions(long companyId, List<Portlet> portlets) {
-		Role powerUserRole = _getPowerUserRole(companyId);
-
-		if (powerUserRole == null) {
-			return;
-		}
-
-		Group userPersonalSiteGroup = _getUserPersonalSiteGroup(companyId);
-
-		if (userPersonalSiteGroup == null) {
-			return;
-		}
-
-		for (Portlet portlet : portlets) {
-			try {
-				if (companyId == portlet.getCompanyId()) {
-					initPermissions(
-						companyId, powerUserRole.getRoleId(),
-						portlet.getRootPortletId(),
-						userPersonalSiteGroup.getGroupId());
-				}
-			}
-			catch (PortalException portalException) {
-				_log.error(
-					StringBundler.concat(
-						"Unable to initialize user personal site permissions ",
-						"for portlet ", portlet.getPortletId(), " in company ",
-						companyId),
-					portalException);
-			}
-		}
-	}
-
-	public void initPermissions(Portlet portlet) {
-		_companyLocalService.forEachCompany(
-			company -> _initPermissions(company, portlet));
-	}
-
-	@Activate
-	protected void activate(BundleContext bundleContext) {
-		_bundleContext = bundleContext;
-
-		String filter = StringBundler.concat(
-			"(&(objectClass=", PanelApp.class.getName(),
-			")(panel.category.key=", PanelCategoryKeys.SITE_ADMINISTRATION,
-			"*))");
```

Comment:
@MrJohn1911 this change can not be right. The original filter here is a "*" wildcard. After your change the wildcard is gone, you won't be collecting the same thing anymore.